testsuite/tests/conftest.py

```diff
@@ -607,15 +609,15 @@ def custom_application(account, custom_app_plan, request, testconfig):  # pylint
         app = custom_application(rawobj.Application("CustomApp", plan))
     """
 
-    def _custom_application(params, autoclean=True, hooks=None, annotate=True):
+    def _custom_application(params, custom_account, autoclean=True, hooks=None, annotate=True):
```

Comment:
Doesn't this break all tests that use `custom_application` directly at they don't include the account?

Reply:
Yes, fixed